format/swf/lite/MovieClip.hx

```diff
@@ -660,14 +660,42 @@ class MovieClip extends flash.display.MovieClip {
 		}
 	}
 
+	private function frame0ChildrenUpdate():Void
+	{
+				var frame = __symbol.frames[0];
+				var remove:Bool = true;
+
+				for( object_id in __objects.keys() )
+				{
+						for (frameObject in frame.objects)
```

Comment:
get on which table? we need to iterate on all instance in __objects, and frame.objects is an array